src/main/java/lotto/model/WinningList.java

```diff
@@ -0,0 +1,41 @@
+package lotto.model;
+
+import static lotto.enumerate.ConfigInteger.LONG_ROUND_DIVIDE_NUMBER;
+import static lotto.enumerate.ConfigInteger.LONG_ROUND_NUMBER;
+import static lotto.enumerate.ConfigString.WINNING_STRING_FIRST;
+
+import java.util.Map.Entry;
+import lotto.enumerate.Rank;
+import lotto.record.AmountRecord;
+import lotto.record.ProfitRate;
+
+public class WinningList {
+
+    private final WinningInfo winningInfo;
+
+    public WinningList(WinningInfo winningInfo) {
+        this.winningInfo = winningInfo;
+    }
+
+    public String printWinningListString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(WINNING_STRING_FIRST.getString());
+        winningInfo.getRankMap().forEach((key, value) -> sb.append(key.getRankString(value)));
+        return sb.toString();
+    }
+
+    public ProfitRate calculateProfitRate(AmountRecord amountRecord) {
+        long sum = winningInfo.getRankMap().entrySet().stream().mapToLong(this::calculateProfit).sum();
```

Comment:
컨벤션에서도 가독성을 위한 스트림 체인의 개행을 허용하고 있으니 참고하시면 좋을 것 같습니다!
개인적으로는 한 줄마다 각 단계의 행동을 구분해서 볼 수 있어서 가독성이 훨씬 좋아진다고 생각합니다
```suggestion
        long sum = winningInfo.getRankMap().entrySet().stream()
                .mapToLong(this::calculateProfit)
                .sum();
```